sdk/include/psdk/winnls.h

```diff
@@ -524,13 +546,35 @@ typedef long LONG_PTR;
 #endif
 
 #if (WINVER >= 0x0600)
-#define MUI_FULL_LANGUAGE             0x01
-#define MUI_LANGUAGE_ID               0x04
-#define MUI_LANGUAGE_NAME             0x08
-#define MUI_MERGE_SYSTEM_FALLBACK     0x10
-#define MUI_MERGE_USER_FALLBACK       0x20
-#define MUI_UI_FALLBACK               MUI_MERGE_SYSTEM_FALLBACK | MUI_MERGE_USER_FALLBACK
-#define MUI_MACHINE_LANGUAGE_SETTINGS 0x400
+#define MUI_FULL_LANGUAGE                   0x0001
+#define MUI_LANGUAGE_ID                     0x0004
+#define MUI_LANGUAGE_NAME                   0x0008
+#define MUI_MERGE_SYSTEM_FALLBACK           0x0010
+#define MUI_MERGE_USER_FALLBACK             0x0020
+#define MUI_UI_FALLBACK                     MUI_MERGE_SYSTEM_FALLBACK | MUI_MERGE_USER_FALLBACK
```

Comment:
Ideally this should be: `(MUI_MERGE_SYSTEM_FALLBACK | MUI_MERGE_USER_FALLBACK)`, within parentheses, but yeaaaahh I see that the official headers don't use that.